src/frontend/js/components/ReferralActivityIndicator/ActivityCloseReferral.tsx

```diff
@@ -0,0 +1,34 @@
+import { DateTime, Duration } from 'luxon';
+import React from 'react';
+import { defineMessages, FormattedMessage } from 'react-intl';
+
+import { GenericErrorMessage } from 'components/GenericErrorMessage';
+import { Spinner } from 'components/Spinner';
+import { useReferral } from 'data';
+import { Referral, ReferralActivityClosed } from 'types';
+
+const messages = defineMessages({
+  loadingReferral: {
+    defaultMessage: 'Loading referral...',
+    description:
+      'Accessible text for the spinner while loading the referral in the activity indicator for urgency change.',
+    id:
+      'components.ReferralActivityIndicator.ActivityCloseReferral.loadingReferral',
+  },
+});
+
+interface ActivityCloseReferralProps {
+  activity: ReferralActivityClosed;
+}
+
+export const ActivityCloseReferral: React.FC<ActivityCloseReferralProps> = ({
+  activity,
+}) => {
+  return (
+    <div>
+      <div className="max-w-md px-2 py-1 rounded border border-gray-400 bg-gray-200">
+        {activity?.message}
```

Comment:
```suggestion
        {activity.message}
```

If the activity is not defined here it's a programmer error.